scripts/publish.sh

```diff
@@ -43,38 +43,41 @@ eu-west-3
 eu-north-1
 "
 
-classic_regions_account_id="906394416424"
+classic_regions_account_id="957584016365"
 
 cn_regions="
 cn-north-1
 cn-northwest-1
 "
 
-cn_regions_account_id="128054284489"
+cn_regions_account_id="957584016365"
 
```

Comment:
Guessing this is left over from testing- account IDs should not change

Reply:
Yes, I will switch these from my personal ID as the last commit before merge.